sparse_strips/vello_cpu/tests/util.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![allow(missing_docs, reason = "we don't need docs for testing")]
+#![allow(
+    unused,
+    reason = "for some reason cargo reports the functions/variables here are unused
+when running `cargo test`, even though they are used."
+)]
```

Comment:
This is because some of the code is unused in certain test cases. That is, this module is compiled at least twice, once for `issues.rs` and once for `basic.rs`.